src/libraries/Microsoft.PowerFx.Core/Public/Types/NamedFormulaType.cs

```diff
@@ -16,6 +16,8 @@ public sealed class NamedFormulaType
 
         public NamedFormulaType(string name, FormulaType type, string displayName = null)
         {
+            Contracts.AssertValue(type);
```

Comment:
>Contracts [](http://example.com/codeflow?start=12&length=9)

nit - this is public method, so we should actually throw ArgumentNullException instead of assert.